### DISCUSSION TITLE
When I run roslaunch command, an RLException error occurs....(Ubuntu)

### DISCUSSION THREAD

**Discussion by Jun-gil** [community/community#45495] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The text is the same as the title.
I tried various methods on google to solve this error. But it doesn't work, so I need your help.
Anyone is welcome, please help me to solve this error.
--------------------------------Command------------------------------------
awesometech@awesometech:~/catkin_ws$ roslaunch urs_wearable world.launch
----------------------------------Result---------------------------------------
... logging to /home/awesometech/.ros/log/165b93fc-9d30-11ed-967c-a99cf3fa13f4/roslaunch-awesometech-4851.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

RLException: while processing /home/awesometech/catkin_ws/src/URSSimEnvironments/urs_wearable/launch/world.launch:
while processing /home/awesometech/catkin_ws/src/URSSimEnvironments/urs_wearable/launch/world.launch:
..
Invalid tag: maximum recursion depth exceeded while calling a Python object.

Arg xml is `<arg name="world" value="$(find urs_wearable)/worlds/test.world"/>`
The traceback for the exception was written to the log file

----------------------------------------------------------------------------------

this is my code I think there is no error in this code.

-------------------------------Code-----------------------------------
`<?xml version="1.0"?>`
`<launch>`
`<include file="$(find urs_wearable)/launch/world.launch">`
`<arg name="world" value="$(find urs_wearable)/worlds/test.world"/>`
`<arg name="reference_latitude" value="32.319939"/>`
`<arg name="reference_longitude" value="-106.763657"/>`
`</include>`
`</launch>`
---------------------------------------------------------------------------------
please help me!!

**Comment by jge162** [comment]
seems like the file urs_wearable/launch/world.launch is including itself and creating an infinite loop,

**Reply by jge162** [reply]
You should go to an ROS forum. Maybe they can guide you more. https://answers.ros.org/questions/

**Reply by Jun-gil** [reply]
Hi Jeremy! Yesterday you gave the same answer. I didn't understand exactly what you said. The recursive call of the world.launch file is not shown in the code I posted. Can you be more precise?

**Reply by jge162** [reply]
The recursion error message you're seeing is being caused by a problem in your launch file. Specifically, it looks like there is an issue with the <arg> tag that is causing the launch file parser to enter into an infinite loop.

I suggest you to check if the package URSSimEnvironments is installed and the folder urs_wearable exists. Also to check if the world test.world exists in the path mentioned in the launch file.

**Reply by Jun-gil** [reply]
URSSimEnvitonmaents is a file I created. So I just converted it to a package. urs_wearable is a package in URSSimEnvitonmaents. The path of test.world is as follows.
urs_wearable/worlds/test.world

Are there any more things left to try?

**Reply by jge162** [reply]
I have no more ideas for you sorry. your issue has to be with files. 

Make sure that the path to your world file is correct. In your message, you mention that the path to the world file is "urs_wearable/worlds/test.world". Verify that this path is correct, and that the file is located at the correct location in your filesystem.

Check if any of the parameter in the launch file is not defined properly

Check if the package URSSimEnvironments is built and sourced correctly.

You can also try to re-build your package using catkin_make command.

**Comment by jge162** [comment]
Please close. Repeat ticket. See original here #45397

**Reply by Jun-gil** [reply]
I'm not familiar with how to use Git. I will use repeat tickets in the future.

**Reply by jge162** [reply]
No worries.
